runtime/queries/fish/highlights.scm

```diff
@@ -109,7 +108,13 @@
   ]
 )
 
-(test_command "test" @function.builtin)
+(command
+  name: (word) @function (#match? @function "^test$")
+  argument: (word) @operator (#match? @operator "^(!?=|-[a-zA-Z]+)$"))
+
+(command
+  name: (word) @punctuation.bracket (#match? @punctuation.bracket "^\\[$")
+  argument: (word) @operator (#match? @operator "^(!?=|-[a-zA-Z]+)$"))
```

Comment:
are these using negative lookaheads? I don't think that's supported by the regex crate

Reply:
It does look like one, but this is actually a plain regex matching the arguments to `test`, which can be `=`, `!=`, `-eq`, etc. I pulled this from https://github.com/ram02z/tree-sitter-fish/blob/master/queries/highlights.scm

And just as a sanity check, it seems to work fine

<img width="283" alt="Screenshot 2025-01-19 at 12 59 26 PM" src="https://github.com/user-attachments/assets/d018f6b1-9657-491a-adb1-3f9abccf76af" />